libs/remote_storage/src/lib.rs

```diff
@@ -111,7 +111,7 @@ pub trait RemoteStorage: Send + Sync + 'static {
 }
 
 pub struct Download {
-    pub download_stream: Pin<Box<dyn io::AsyncRead + Unpin + Send>>,
+    pub download_stream: Pin<Box<dyn io::AsyncRead + Unpin + Send + Sync>>,
```

Comment:
I suppose it's referenced somewhere. &Send is !Sync.

Reply:
Yes. Safekeeper walsender became async, `WalReader` containing such stream (due to taking WAL from s3) is now held across awaits and so it has to be Sync since futures can be moved between threads of the runtime.